### DISCUSSION TITLE
Form Validation

### DISCUSSION THREAD

**Discussion by Lahali** [IT-Academy-BCN/ita-wiki#49] [General]
@vicenlu 
Para validar el form de registro, entendemos que todos los campos son obligatorios, ¿cómo se lo enseñamos al usuario? Por un lado se puede poner un asterisco con una nota pequeña abajo de todo, pero también podemos: 
- poner los inputs con color error hasta que esté todo relleno,
- dejar el botón como disabled hasta que esté todo relleno. 

¿Qué opción preferís?

**Comment by kevinmamaqi** [comment]
Creo que ya esta respondido.
